nephos/fabric/ord.py

```diff
@@ -71,31 +71,32 @@ def setup_ord(opts, upgrade=False):
         opts (dict): Nephos options dict.
         upgrade (bool): Do we upgrade the deployment? False by default.
     """
-    ord_namespace = get_namespace(opts, opts["orderers"]["msp"])
+    ord_namespace = get_namespace(opts, msp="AlphaMSP")
     # Kafka
-    if "kafka" in opts["orderers"]:
+    if "kafka" in opts["msps"]["AlphaMSP"]["orderers"]:
         # Kafka upgrade is risky, so we disallow it by default
         version = get_version(opts, "kafka")
-        config_yaml = f"{opts['core']['dir_values']}/kafka/{opts['orderers']['kafka']['name']}.yaml"
+        kafka_config = get_kafa_configs(opts=opts)
+        config_yaml = f"{opts['core']['dir_values']}/AlphaMSP/kafka/{kafka_config['name']}.yaml"
         extra_vars = helm_extra_vars(version=version, config_yaml=config_yaml)
         helm_install(
             "incubator",
             "kafka",
-            opts["orderers"]["kafka"]["name"],
+            kafka_config['name'],
```

Comment:
Let's run Python "black" on the repository, to have consistent quotations formatting.

Reply:
We can do this in a separate PR.